schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+                            "Subtitle",
+                            "TranslatedTitle"
+                        ]
+                    }
+                },
+                "required": [
+                    "title"
+                ]
+            }
+        },
+        "container": {
+            "description": "The container of the resource.",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "The identifier for the container.",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "The type of the container.",
+                    "type": "string",
+                    "enum": [
+                        "Book",
+                        "BookSeries",
+                        "Journal",
+                        "Periodical",
+                        "ProceedingsSeries",
+                        "Series"
+                    ]
+                },
+                "title": {
+                    "description": "The title of the container.",
+                    "type": "string"
+                }
+            }
+        },
+        "contributors": {
+            "description": "The contributors to the resource.",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "description": "The unique identifier for the contributor.",
+                        "type": "string"
+                    },
+                    "type": {
+                        "description": "The type of the contributor.",
+                        "type": "string",
+                        "enum": [
+                            "Organization",
+                            "Person"
+                        ]
+                    },
+                    "name": {
+                        "description": "The name of the contributor.",
+                        "type": "string"
+                    },
+                    "givenName": {
+                        "description": "The given name of the contributor.",
+                        "type": "string"
+                    },
+                    "familyName": {
+                        "description": "The family name of the contributor.",
+                        "type": "string"
+                    },
+                    "affiliation": {
+                        "$ref": "#/definitions/affiliations"
+                    }
+                }
+            }
+        },
+        "subjects": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "subject": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "subject"
+                ]
+            },
+            "uniqueItems": true
+        },
```

Comment:
Do you have any kind of controlled vocabulary that you're using for this?

Reply:
This is historically a very messy topic. As a starting point I started using the OECD Fields of Science and Technology with DataCite, and I still think that this is a good high level subject classification (less than 50 categories). I also like the Australian/NZ classification ANZSRC https://www.abs.gov.au/statistics/classifications/australian-and-new-zealand-standard-research-classification-anzsrc/latest-release which Figshare and Dimensions and of course lots of people in AUS and NZ use.